README.md

```diff
@@ -177,6 +177,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Nagarjun Software Technosys Pvt. Ltd.](https://nagarjunsofttech.com/) | Nagarjun, Kathmandu | Online Marketing and Mobile Apps Software Development
 - [Naxa Pvt. Ltd.](https://naxa.com.np/) | Maharajgunj, Kathmandu | Address contemporary problems in developing countries through technological innovations, particularly in digital and spatial technology
 - [Neem Infosys](https://neeminfosys.com/) | Mahendranagar, Kanchanpur | We develop user-centric website, mobile apps and digital marketing that grows startup and existing businesses in the sales and service industries
+- [Nepal Digital Systems Pvt. Ltd.](https://nepaldigisys.com/) | Dhobighat, Lalitpur | A company working on IoT and Embedded Systems
```

Comment:
@lomassubedi Thank you for contributing.

The description is a little too generic. Can we update it?